cocos/network/HttpAsynConnection.m

```diff
@@ -180,8 +180,13 @@ - (BOOL) shouldTrustProtectionSpace:(NSURLProtectionSpace*)protectionSpace
         SecTrustSetExceptions(serverTrust, errDataRef);
         
         SecTrustEvaluate(serverTrust, &trustResult);
+        [(id)errDataRef release];
+        
     }
     
+    [certData release];
+    [(id)certArrayRef release];
```

Comment:
@hobogenized ,I have create a new pr to resolve the memory leak for AudioEngine.
Thanks for contributing and could you help to review these codes.
#12103.